cypress/integration/recordings.spec.js

```diff
@@ -15,23 +20,20 @@ context("Recordings", function () {
   });
 
   describe("On the definition page", () => {
-    it("should play a recording via a 🔊 icon", function () {
-      cy.fixture("recording/_search/wâpamêw", "utf-8").as("recordingsResults");
+    beforeEach(() => {
+      // Intercept calls to our API
+      cy.intercept("https://speech-db.altlab.app/api/bulk_search?*", {
+        fixture: "recording/bulk_search/wâpamêw.json",
+      }).as("recordingsResults");
+    });
 
+    it("should play a recording via a 🔊 icon", function () {
       // Get to the definition/paradigm page for "wâpamêw"
-      cy.visitSearch("wâpamêw");
-      cy.contains("a", "wâpamêw").click();
-      cy.url().should("contain", "/word/");
-
-      // TODO: we should stub a network request,
-      // but Cypress can't deal with fetch() requests :/
-      // It has to use a polyfill, but for various reasons,
-      // that's annoying and brittle
-      //  (e.g., if the URL changes, the polyfill needs
-      //   to be re-enabled).
```

Comment:
The Cypress version we were using originally did not support stubbing these requests, however, the current version has `cy.intercept()` that _does_ work with `fetch()` requests!